api/lib/utils/metricsCollectorUtils.js

```diff
@@ -14,6 +14,18 @@ module.exports = function(metricsCollectorSettings) {
       timeout: 10000,
     }
     if (metricsCollectorSettings.tls) {
+      if(!fs.existsSync(metricsCollectorSettings.tls.keyFile)){
+        logger.error("Invalid TLS key path: " + metricsCollectorSettings.tls.keyFile);
+        throw new Error("Invalid TLS key path: " + metricsCollectorSettings.tls.keyFile);
+      }
+      if(!fs.existsSync(metricsCollectorSettings.tls.certFile)){
+        logger.error("Invalid TLS certificate path: " + metricsCollectorSettings.tls.certFile);
+        throw new Error("Invalid TLS certificate path: " + metricsCollectorSettings.tls.certFile);
+      }
+      if(!fs.existsSync(metricsCollectorSettings.tls.caCertFile)){
+        logger.error("Invalid TLS ca certificate path: " + metricsCollectorSettings.tls.caCertFile);
+        throw new Error("Invalid TLS ca certificate path: " + metricsCollectorSettings.tls.caCertFile);
+      }
```

Comment:
where is the relevant test case?

Reply:
I removed this from this PR. will resubmit later